person.rb

```diff
@@ -1,18 +1,16 @@
 require_relative './nameable'
-require_relative './base_decorator'
-require_relative './capitalize_decorator'
-require_relative './trimmer_decorator'
 
 class Person < Nameable
   attr_accessor :age, :name
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @age = age
     @name = name
     @parent_permission = parent_permission
     @id = Random.rand(1..10_000).to_s
+    @rentals = []
   end
 
```

Comment:
- [ ] To implement the `has-many` side of the relationship(a personhas many rentals), a method for adding rentals should be present. Something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(book, self, date)
  end
```
Checkout this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) and kindly fix that.